prover/README.md

```diff
@@ -24,18 +24,24 @@ source ./scripts/dev_setup.sh
 The prover now works with a default training wheel key pair (already prepared at `private_key_for_testing.txt`)
 and optionally a "next" one (already prepared at `private_key_for_testing_another.txt`).
 
-The prover now works with a default circuit (prepared by `dev_setup` at `~/.local/share/aptos-prover-service/setup_2024_05`)
+The prover now works with a default circuit (prepared by `dev_setup` at `~/.local/share/aptos-prover-service/default`)
 and optionally a "next" one (prepared by `dev_setup` at `~/.local/share/aptos-prover-service/setup_initial`).
```

Comment:
Shouldn't the directory be called `next` or something? `setup_initial` seems like a confusing name here.

Reply:
ah i forgot to update L28. it's called "new" now.